go/vt/vtorc/inst/instance_dao.go

```diff
@@ -1210,3 +1210,20 @@ func ExpireStaleInstanceBinlogCoordinates() error {
 	}
 	return ExecDBWriteFunc(writeFunc)
 }
+
+// SnapshotDatabaseState takes the snapshot of the database and returns it.
+func SnapshotDatabaseState() (string, error) {
```

Comment:
Awfully nitty, but a better name IMO would be `GetDatabaseState`.

Reply:
Changed